src/shared/telemetry/exemptMetrics.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * TODO: Fix calls to these metrics and remove this file.
+ * NOTE: Do NOT add additional metrics here, your PR will be rejected. All new metrics should pass validation.
+ *
+ * Checks were added to all emitted telemetry metrics to ensure they contain the correct properties.
+ * This check does not exist outside test/automated runtimes because the error is not useful
+ * for the user and doesn't break telemetry.
+ *
+ * These are metrics that fail validation and are emitted by code that is covered in test cases.
+ * This allowlist exists to fix these incrementally and not have CI fail in the meantime.
+ */
+const validationExemptMetrics: Set<string> = new Set([
+    'amazonq_isReviewedChanges',
+    'apigateway_copyUrl',
```

Comment:
Can we also fix a couple of these in this PR? That adds some useful context to the changeset.

Reply:
Would like to leave the fixes out of scope for this PR, but I can see some value in context. Will fix a metric.